src/plugins.py

```diff
@@ -0,0 +1,28 @@
+# Plugin loader
+#
+# Modular plugin loader
+import importlib
+
+preloaded_plugins = []
+
+
+def load(config):
+    plugins_available = config["plugins"]
+
+    for plugin in plugins_available:
+        plugin_key = list(plugin.keys())[0]
```

Comment:
Not particularly safe. Assumes that `plugin.keys().length > 0`. Also, isn't there a better way to do this?